translations/frontend/de.json

```diff
@@ -619,6 +619,8 @@
                 "media_play": "Abspielen",
                 "media_play_pause": "Spielen/Pause",
                 "media_previous_track": "Vorheriger Titel",
+                "media_seek_backward": "Zurückspulen",
+                "media_seek_forward": "Vorspulen",
```

Comment:
This has to be removed, since translations are not done in the code directly. See the HA translation documentation.